src/metabase/models/card.clj

```diff
@@ -227,13 +229,24 @@
      (params/assert-valid-parameter-mappings card)
      (collection/check-collection-namespace Card (:collection_id card)))))
 
+(defn- tokenize-dataset-query [card]
+  (when (and (= (mdb/db-type) :postgres)
+             (= (:query_type card) :native))
+    (db/execute! {:update Card
+                  :set    {:dataset_query_tokens (hsql/raw "to_tsvector(cast(dataset_query as json) -> 'native' -> 'query'")}
```

Comment:
isn't it `plainto_tsvector` for arbitrary user stuff? otherwise you could shove tsquery operators in there or something

Reply:
`plainto_tsvector` doesn't exist. I think you're thinking about `plainto_tsquery`.